docs/source/tutorial/tips_faqs.rst

```diff
@@ -39,3 +39,38 @@ Please refer to `MPI4py API reference <http://pythonhosted.org/mpi4py/apiref/mpi
 Using FP16
 ~~~~~~~~~~
 FP16 (16-bit half precision floating point values) is not supported in ChainerMN as of now.
```

Comment:
This faq is duplicate with #249 and will conflict with it.

Reply:
Fixed!